lib/brains/management/commands/braintree_config.py

```diff
@@ -40,23 +48,23 @@ def product_exists(plans, external_id, amount):
     Check that the product exists in Braintree.
     """
     if external_id not in plans:
-        log.warning(
-            'The plan: {0} does not exist in Braintree. Plans cannot be '
-            'created from the API and must be created from the Braintree '
-            'console.'
-            .format(external_id))
-
-        raise CommandError('Missing product: {0}'.format(external_id))
+        raise BraintreePlanDoesNotExist(
```

Comment:
+1 for more explicit errors, would move them into https://github.com/mozilla/solitude/blob/master/lib/brains/errors.py and import them though.